optuna_dashboard/ts/components/GraphHistories.tsx

```diff
@@ -0,0 +1,259 @@
+import * as plotly from "plotly.js-dist-min"
+import React, { ChangeEvent, FC, useEffect, useState } from "react"
+import {
+  Grid,
+  FormControl,
+  FormLabel,
+  FormControlLabel,
+  MenuItem,
+  Typography,
+  SelectChangeEvent,
+  Select,
+  Radio,
+  RadioGroup,
+  useTheme,
+} from "@mui/material"
+
+import { plotlyDarkTemplate } from "./PlotlyDarkMode"
+import {
+  getFilteredTrials,
+  Target,
+  useObjectiveAndUserAttrTargets,
+} from "../trialFilter"
+
+const plotDomId = "graph-histories"
+
+interface HistoryPlotInfo {
+  study_name: string
+  trials: Trial[]
+  directions: StudyDirection[]
+}
+
+export const GraphHistories: FC<{
+  studies: StudyDetail[]
+  logScale: boolean
+  includePruned: boolean
+}> = ({ studies, logScale, includePruned }) => {
```

Comment:
Do you think we can unify this component with GraphHistory?

Reply:
Unified.